services/supervisor/src/lib.rs

```diff
@@ -12,6 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Supervisor is an [Exonum][exonum] service capable of the following activities:
+//! - Service artifact deployment;
+//! - Service instances creation;
+//! - Changing consensus configuration;
+//! - Changing service instances configuration.
+//!
+//! More information on the artifact/service lifecycle can be found at the Exonum
+//! [runtime module documentation][runtime-docs].
+//!
+//! Supervisor service has two different operating modes: "simple" and "decentralized".
+//! The difference between modes is in the decision making approach:
+//! - Within decentralized mode, to deploy a service or apply a new configuration, there should
+//!   be no less than (2/3)+1 validator approvals for a decision;
+//! - Within simple mode, any decision will be executed after just 1 validator approval.
+//!
+//! Simple mode can be useful if all the validator nodes are belong to the one network
+//! administrator, or for development purposes.
+//! For a network with a low node confidence, consider using decentralized mode.
+//!
+//! **Interaction:**
+//!
+//! On a transaction level, every request should be signed by a validator **node**. Thus, one is
```

Comment:
```suggestion
//! On a transaction level, every request should be signed with a key of validator **node**. Thus, one is
```
???

Reply:
Well, not sure if suggested variant is better. Node signs the transaction itself and broadcasts it.

Reply:
And this is kinda the key point here: user **should not** send request as transactions by himself, since the **node** will create and sign it.